portal-web/docroot/html/taglib/aui/nav_item/end.jsp

```diff
@@ -31,10 +31,10 @@ if (Validator.isNull(title)) {
 %>
 
 <c:if test="<%= !dropdown || Validator.isNotNull(bodyContentString.trim()) %>">
-	<li class="<%= cssClass %><%= selected ? " active " : StringPool.SPACE %><%= state %>" id="<%= id %>" role="presentation" <%= AUIUtil.buildData(data) %> <%= InlineUtil.buildDynamicAttributes(dynamicAttributes) %>>
+	<li class="<%= cssClass %><%= !dropdown ? " nav-item " : "" %><%= selected ? " active " : StringPool.SPACE %><%= state %>" id="<%= id %>" role="presentation" <%= AUIUtil.buildData(data) %> <%= InlineUtil.buildDynamicAttributes(dynamicAttributes) %>>
```

Comment:
We need to move `<%= selected ? " active " : StringPool.SPACE %>` to `nav-link`.